### DISCUSSION TITLE
Try to download an installation file

### DISCUSSION THREAD

**Discussion by BjH14** [qarmin/czkawka#1393] [Q&A]
Hello,
I would like to download and install (on Windows 11) czkawka on the page of the last release : 
https://github.com/qarmin/czkawka/releases/tag/8.0.0
I don't know which file or folder I have to use.
I'm new on Github.
Thank you for your help

**Comment by harshness** [comment]
windows_czkawka_gui_410.zip

This is the standard Windows GTK GUI version.  Note that you may have to click a "show more files" button to see the Windows binaries.

**Reply by rgc1210** [reply]
Just found this and was hoping this would solve my dups problems but downloaded the GTK GUI version mentioned, unzipped and see two exe files, clicking on them does nothing so how do I launch this on Win11?
